src/components/dashjs-statistics/dashjs-statistics.tsx

```diff
@@ -1,296 +1,416 @@
-import { Component, Host, h, Prop, Watch, Element, State} from '@stencil/core';
+import { Component, Host, h, Prop, Watch, Element, State, Listen } from '@stencil/core';
 import * as chartjs from 'chart.js';
 const { Chart } = chartjs.default.Chart;
 
-
 @Component({
   tag: 'dashjs-statistics',
   styleUrl: 'dashjs-statistics.css',
   shadow: false,
 })
-
 export class DashjsStatistics {
+  @Prop()
+  video_data: any;
+  @Prop()
+  audio_data: any;
+
+  @State()
+  videoDisable: string = 'Disable';
+
+  @State()
+  audioDisable: string = 'Disable';
+
+  // DashMetrics properties
+  @State()
+  videoBufferLevel: number;
```

Comment:
those metrics might change, should i generate a JSON with all metrics from the documentation so you can do this on a higher level, without writing down every single metric?

Reply:
But I would still need to declare a state for each right? As some function are from `dashMetrics` and `dashAdapter` and each one takes different parameters as you can see in `streamMetrics()`

Reply:
no, more like a map of these values

Reply:
Okay👍🏼

Reply:
if you want to incorporate it now just use an array with those properties as strings... then I can plug in the generated JSON later.